frontend/webapp/configs/config.py

```diff
@@ -74,7 +74,7 @@ def configure_app(app):
     app.config.from_object("webapp.configs.config.BaseConfig")
     app.config.from_pyfile("/etc/artemis/webapp.cfg", silent=False)
 
-    app.artemis_logger = logging.getLogger("webapp_logger")
+    app.artemis_logger = logging.getLogger("artemis_logger")
```

Comment:
also needs to be changed on local_configs. Are you sure we need this change? Why can't we keep it as webapp_logger and avoid all potential issues with users forgetting to change this locally (on non-git-controlled files)?

Reply:
I decided to do it this way to keep consistency. Other logging yaml files are artemis_logger and also the function that actually prints is artemis_logger as well

Reply:
but I can keep it the old way if we want to have this syntax

Reply:
It is probably OK, but put a reminder somewhere to warn people to change their logging.yaml files for the frontend (and config.py), if they upgrade to the new version.